### ISSUE TITLE
JAWS announcing the Web component “Edit combo” as “list box” instead of edit combo

### ISSUE THREAD

**Issue by Rasubra8** [FreedomScientific/VFO-standards-support#502]
### Summary
JAWS does not read the appropriate role for Web component Edit combobox 
> Brief description of the issue. Please list any specific steps.
In the following link
https://carbon-web-components.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/582/iframe.html?id=components-combo-box--default&knob-Helper%20text%20%28helper-text%29=Optional%20helper%20text&knob-Label%20text%20%28label-text%29=test&knob-The%20placeholder%20content%20%28trigger-content%29=Filter...&knob-The%20validity%20message%20%28validity-message%29=&knob-The%20value%20of%20the%20selected%20item%20%28value%29=&viewMode=story

1.	JAWS incorrectly identifies the Edit combo box as listbox in Arrow navigation. In Tab/shift tab navigation only “has-popup” is announced. 
2.	After making a selection, the selected value is not announced. 
As per the dev it (Still does not work After trying with aria-selected, aria-activedescendant , and giving IDs to the options). But working fine in Chrome.
Note: This preview link may not work after 60 days due to the deployment settings
Could this be a jaws issue in FF?

### Expected result
JAWS should announce the "edit combo" role in both arrow and tab navigation.And the selected value should be announced After making the selection.
> What did you expect to happen?
JAWS should announce the "edit combo" role in both arrow and tab navigation.And the selected value should be announced After making the selection.

### Actual result

> What actually did happen?
JAWS does not read the appropriate role and the selected value 
### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

#### JAWS version and build number
JAWS 2020.2012.13

 
#### Operating System and version
win10

#### Browser and version:
FF86

**Comment by JAWS-test** [comment]
The source code seems so wrong that I am amazed that anything is output at all. For example, the combobox is marked with aria-hidden=true and around the combobox is a listbox

**Comment by stevefaulkner** [comment]
In Firefox it is exposed in the accessibility tree as a listbox
